.cr/personal/FavoritesList/List.xml

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Root Type="DevExpress.CodeRush.Foundation.CodePlaces.Options.FavoritesListContainer">
```

Comment:
not sure what this file is?

Reply:
Cant find this file lol, exclude it if you are able to please.